packages/react/src/experimental/OneDataCollection/useSelectable.ts

```diff
@@ -66,6 +68,32 @@ export function useSelectable<
     new Map()
   )
 
+  /**
+   * Set the default selected items and groups
+   */
+  useEffect(() => {
+    if (defaultSelectedItems) {
+      if (isGrouped) {
+        ;(defaultSelectedItems.groups || []).forEach((defaultGroup) => {
```

Comment:
what do you think on using `for..of` loop here? I prefer if over callback driven `forEach` for readability. Also it will eliminate the need of `;`

Reply:
sure, and for..of is faster that foreach